City-of-Helsinki/tilavarauspalvelu-ui
# Adds short day of the week to the date, above my-units calendar

## 🛠️ Changelog
- Adds the short day name next to the date, above the calendar in my-units section (admin side)
- Also attempts to add the same favicon which is in use on the ui/client-side, in exactly the same way - however it doesn't seem to have intended consequences (..or any at all 😮)

## 🧪 Test plan
- Check out the calendar in a reservation unit page, under my units.
- See if anything sticks out to you regarding why the favicon isn't showing as on the UI-side

## 🎫 Tickets

- TILA-3010
